### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by fredrikekre**

### CODE REVIEW

File: test/test_ops.jl [hunk 1]
```diff
@@ -252,44 +228,11 @@ for dim in (1,2,3)
         ad2 = Vec{3}((0.0,1.0,0.0))
         @test ad × ad2 ≈ Vec{3}((0.0, 0.0, 1.0))
     end
+end # of testset
 
-    ##########################
-    # Creating with function #
-    ##########################
```
Comment (KristofferC) [inline test/test_ops.jl:258]: what was bad with this stuff? :P

Comment (fredrikekre) [reply test/test_ops.jl:258]: Just moved it. Didn’t quite fit in the tensor operations test set :)

Comment (KristofferC) [reply test/test_ops.jl:258]: Oh, okay, cool! Merge when ready,

**Comment by fredrikekre** [comment]
Btw, is there a way to gather inner testsets? What i want to happen here is that there is only one inner testset, like in https://github.com/KristofferC/TimerOutputs.jl where there is a look up for the same description.

Test case:

``` jl
@testset "outer" begin
    for i in 1:3
        @testset "first inner" begin
            @test 1 == i
        end

        @testset "second inner" begin
            @test 1 == i
        end
    end
end
```

results in:

``` jl
Test Summary:  | Pass  Fail  Total
  outer        |    2     4      6
  first inner  |    1            1
  second inner |    1            1
  first inner  |          1      1
  second inner |          1      1
  first inner  |          1      1
  second inner |          1      1
```

but it would be better with:

``` jl
Test Summary:    | Pass  Fail  Total
  outer          |    2     4      6
    first inner  |    1     2      3
    second inner |    1     2      3
```

**Comment by KristofferC** [comment]
I agree it is better. Don't think it is possible with current Base test system though.

**Pull request merged**